src/sources/shopify/tracker/transform.js

```diff
@@ -142,7 +143,7 @@ const createMessageForEvent = (event, shopifyTopic) => {
 
 const handleCartsUpdate = async (event, metricMetadata) => {
   if (!useRedisDatabase) {
-    return { message: trackPayloadBuilder(event, 'carts_update') };
+    return { redisData: undefined };
```

Comment:
this change is fine to do?

Reply:
you need to confirm that, I am just shuffling the code but it should have same effect